tests/integration-tests/tests/schedulers/test_slurm_accounting/resources/get_accounting_users.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+DEFAULT_COOKBOOK_ATTRIBUTES_FILE="/etc/chef/cookbooks/aws-parallelcluster/attributes/default.rb"
+
+get_attribute_value() {
+  local _file=$1
+  local _search=$2
+  local _pattern
+
+  _pattern=$(grep -F "default${_search}" "${_file}")
+  echo ${_pattern} | tr -d '\n' | cut -d = -f 2 | xargs
+}
```

Comment:
I we decide to search the `SlurmUser` in the Slurm configuration, I guess we can dismiss this.

Reply:
Changed to get SlurmUser form slurm.conf